pkg/apis/pipeline/v1beta1/taskrun_conversion.go

```diff
@@ -193,6 +218,156 @@ func (trso *TaskRunSidecarOverride) convertFrom(ctx context.Context, source v1.T
 	trso.Resources = source.ComputeResources
 }
 
+// ConvertTo implements apis.Convertible
+func (trs *TaskRunStatus) ConvertTo(ctx context.Context, sink *v1.TaskRunStatus) error {
+	// TODO: handle ducktype conversion once having a fix at https://github.com/tektoncd/pipeline/pull/5794#discussion_r1031827225
```

Comment:
This TODO is a bit confusing. Is there an upstream issue that should be linked to here?

Reply:
Updated as // TODO: handle ducktype conversion using knative func once https://github.com/knative/pkg/issues/2655 is fixed